specification/network/resource-manager/Microsoft.Network/stable/2018-08-01/virtualNetwork.json

```diff
@@ -911,6 +942,13 @@
           },
           "description": "Gets an array of references to the external resources using subnet."
         },
+        "delegations": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Delegation"
+          },
+          "description": "Gets an array of references to the delegations on the subnet."
+        },
         "provisioningState": {
```

Comment:
You need to include the "purpose" property as a read-only property or the SDK (and thus CLI) will not see it.